src/flash-source-buffer.js

```diff
@@ -23,6 +25,17 @@ const scheduleTick = function(func) {
 };
 
 /**
+ * Generates a random string of length 10
+ *
+ * @return {String} the randomly generated string
+ * @function generateRandomString
+ * @private
+ */
+const generateRandomString = function() {
+  return (Math.random().toString(36) + '00000000000000000').slice(2, 8);
```

Comment:
This is a good point. The 0s help to guarantee that a shorter string won't be returned. But we don't really care about string length, just having a random string. 

(As for the 17 zeros, forgive me for I have copy pasted from stackoverflow)